lib/chef-dk/skeletons/code_generator/recipes/build_cookbook.rb

```diff
@@ -156,8 +157,8 @@
     only_if "git status --porcelain |grep \".\""
   end
 
-  execute("git-return-to-master-branch") do
-    command("git checkout master")
+  execute("git-return-to-#{pipeline}-branch") do
+    command("git checkout #{pipeline}")
```

Comment:
What does `pipeline` evaluate to here? Is `pipeline` equivalent to `add-delivery-configuration`? If so, I think that's why all the tests are failing. Previously, we returned to master and merged and then deleted, now we're returning to `pipeline` and merging and deleting, but the delete is failing because we're on `add-delivery-configuration`.